Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -742,20 +731,19 @@ function vtkOpenGLImageMapper(publicAPI, model) {
           pwfun = actorProperty.getPiecewiseFunction(c);
           if (pwfun === null) {
             // Piecewise constant max if no function supplied for this component
-            pwfFloatTable.fill(255.0);
+            pwfFloatTable.fill(1.0);
           } else {
             const pwfRange = pwfun.getRange();
             pwfun.getTable(pwfRange[0], pwfRange[1], pwfWidth, tmpTable, 1);
             // adjust for sample distance etc
             if (iComps) {
               for (let i = 0; i < pwfWidth; i++) {
-                pwfFloatTable[c * pwfWidth * 2 + i] = 255.0 * tmpTable[i];
-                pwfFloatTable[c * pwfWidth * 2 + i + pwfWidth] =
-                  255.0 * tmpTable[i];
+                pwfFloatTable[c * pwfWidth * 2 + i] = tmpTable[i];
```

Comment:
@thewtex These do need to be in the range `[0.0, 1.0]`, rather than `[0.0, 255.0]` in order to be used as opacity values in the case of dependent components.   The reason it was working for independent components, even though they weren't in that range, is due to how we normalize them per-fragment anyway.

Reply:
@scottwittenburg :+1: :1st_place_medal: